velox/exec/HashBuild.cpp

```diff
@@ -45,6 +45,8 @@ BlockingReason fromStateToBlockingReason(HashBuild::State state) {
       VELOX_UNREACHABLE(HashBuild::stateName(state));
   }
 }
+
+const std::string kJoinTableBuildWallNanos{"hashtable.buildWallNanos"};
```

Comment:
Naming and consistency issues still remain.

> kJoinTableBuildWallNanos

This runtime stat name uses a constant, while other stat names are hardcoded. It would be nice to not hard-code stat names and use constants for these, but this needs to be done consistently for all of them.

This constant name includes 'Join' redundantly since it appears in the context of the file that only applies to joins.